redaxo/src/addons/structure/lib/navigation.php

```diff
@@ -48,12 +52,12 @@ class rex_navigation
 
     private function __construct()
     {
-        // nichts zu tun
+        $this->activeLiClass = $this->activeClass;
+        $this->activeAClass = $this->activeClass;
+        $this->currentLiClass = $this->activeClass;
+        $this->currentAClass = $this->activeClass;
```

Comment:
Nein, passt alles. Ich verstehe gerade nur noch meinen Denkfehler nicht. Die Setter funktion ersetzt doch $this->wasauchimmer und _getNavigation() bezieht sich doch darauf.